src/main/java/slimeknights/tconstruct/library/fluid/FluidTankBase.java

```diff
@@ -0,0 +1,31 @@
+package slimeknights.tconstruct.library.fluid;
+
+import net.minecraft.world.World;
+import net.minecraft.world.server.ServerWorld;
+import net.minecraftforge.fluids.capability.templates.FluidTank;
+import slimeknights.mantle.tileentity.MantleTileEntity;
+import slimeknights.tconstruct.library.network.TinkerNetwork;
+import slimeknights.tconstruct.smeltery.network.FluidUpdatePacket;
+
+public class FluidTankBase<T extends MantleTileEntity> extends FluidTank {
```

Comment:
What does `MantleTileEntity` provide these days? I know I used to have a `markDirtyFast` there as I needed it here, is that still present? If not, switch to just plain `TileEntity` (or consider implementing `markDirtyFast` again)

Reply:
Honestly not sure if still needed. Just has markDirtyFast and an
'isClient' utility.